package-lock.json

```diff
@@ -0,0 +1,1814 @@
+{
```

Comment:
Don't commit the package-lock.json, preferably would add it to .gitignore

Reply:
Fixed